utils/build/docker/java/spring-boot/pom.xml

```diff
@@ -434,5 +439,6 @@
         <protobuf.version>3.16.3</protobuf.version>
         <grpc.version>1.31.0</grpc.version>
         <packaging.type>jar</packaging.type>
+        <dd-trace-java.version>1.35.0</dd-trace-java.version>
```

Comment:
Does this work with previous versions of the tracer? We might need to baseline this to an older API version.

Reply:
> Does this work with previous versions of the tracer? We might need to baseline this to an older API version.

It will most likely work with the first version of the tracer including the SDK, I'll try to use find the first one where this works.

Reply:
Just removed the dependency, it was already defined in the tracer and I completely missed it: